src/main/java/com/cloudant/http/HttpConnection.java

```diff
@@ -68,6 +67,23 @@
 public class HttpConnection {
 
     private static final Logger logger = Logger.getLogger(HttpConnection.class.getCanonicalName());
+
+    private static final String USER_AGENT;
+
+    static {
+        String ua = "java-cloudant-http/unknown";
```

Comment:
OK, perhaps getUserAgent is better. This means we might have some duplicated code for getting platform versions etc.